pages/common/git-for-each-repo.md

```diff
@@ -0,0 +1,12 @@
+# git for-each-repo
+
+> Run a Git command on a list of repositories.
+> This command is experimental. **The behavior may change**.
```

Comment:
```suggestion
> This command is experimental. **The behavior may change**.
> More information: <https://git-scm.com/docs/git-for-each-repo>.
```

Reply:
OMG, I looked that page and I didn't find it.